kotlin-jvm/build.gradle.kts

```diff
@@ -1,46 +1,25 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 plugins {
     kotlin("jvm")
-    kotlin("kapt")
-    kotlin("plugin.serialization")
-    id("com.squareup.sqldelight")
-    id("com.apollographql.apollo")
+    // NOTE: IF the library does code generation, either with kapt or a gradle/compiler plugin
+    // NOTE: THEN it belongs in "kotlin-codegen", not here!
```

Comment:
This would need to be changed to separate slow from fast codegen.

Reply:
done